kdf.c

```diff
@@ -89,9 +71,7 @@ void kdf_login_key(const char *username, const char *password, int iterations, c
 		iterations = 1;
 
 	if (iterations == 1) {
```

Comment:
Consider defining a MACRO for the minimum number of iterations according to what is already enforced on other LastPass clients and changing this if statement to compare against that, thus, enforcing the same minimum number of iterations for all clients. Example: 

```
#define MINIMUM_ITERATIONS 600000

if (iterations < MINIMUM_ITERATIONS) {
  // foo
}
```